website/pages/docs/from-1.5/path-variables.mdx

```diff
@@ -12,3 +12,31 @@ description: |-
 - `path.cwd`: the directory from where Packer was started.
 
 - `path.root`: the directory of the input HCL file or the input folder.
+
+
+These can be used in the following ways:
```

Comment:
```suggestion
## Examples
```
I think the use of the Examples heading on other variable pages help draw the user to the correct place.

Reply:
Sounds good to me; thanks !